sdcm/stress/latte_thread.py

```diff
@@ -162,21 +162,24 @@ def parse_final_output(result):
         :param result: output of latte stats
         :return: dict
         """
-        ops_regex = re.compile(r'Throughput(.*?)\[op/s\]\s*(?P<op_rate>\d*)\s')
-        latency_99_regex = re.compile(r'\s*99\s*(?P<latency_99th_percentile>\d*\.\d*)\s')
-        latency_mean_regex = re.compile(r'\s*Mean resp. time\s.*\s(?P<latency_mean>\d*\.\d*)\s')
+        ops_regex = re.compile(r'\s*Throughput(.*?)\[op\/s\]\s*(?P<op_rate>\d*)\s')
+        latency_99_regex = re.compile(r'\s* 99 \s*(?P<latency_99th_percentile>\d*\.\d*)\s')
+        latency_mean_regex = re.compile(r'\s*Mean resp. time\s*(?:\[(ms|s)\])?\s*(?P<latency_mean>\d+\.\d+)')
```

Comment:
maybe we can extend the integration test to cover we parse those right ?

Reply:
Actually, some existing tests must fail. So, I need to update it.
Also, looks like integration tests are broken after adding of the `LatencyCalculatorMixedResult`.